Shared/Samples/Apply colormap renderer to raster/ApplyColormapRendererToRasterView.swift

```diff
@@ -19,7 +19,7 @@ struct ApplyColormapRendererToRasterView: View {
     /// Creates the raster layer used by this sample.
     static func makeRasterLayer() -> RasterLayer {
         // Creates a raster and raster layer.
-        let raster = Raster(name: "ShastaBW", extension: "tif", bundle: nil)!
+        let raster = Raster(name: "ShastaBW/ShastaBW", extension: "tif", bundle: nil)!
```

Comment:
There is a piece of internal logic in `Bundle+FileManager.swift` that recursively search  for the resource in the bundle, so this shouldn't be needed. However, it currently crashes on TestFlight build without this fix, so I'm not sure why…